source/install/wizard.rst

```diff
@@ -0,0 +1,148 @@
+Install wizard
+==============
+
+To begin installation process, point your browser to the GLPI main address:
+`https://{adresse_glpi}/ <https://{adresse_glpi}/>`_
+
+When GLPI is not installed; a step-by-step installation process begins.
+
+Choose lang (Select your language)
+----------------------------------
+
+The first step will let you choose the installation language. Select your lang, and click validate.
+
+.. image:: images/select_language.png
+   :alt: Choose lang
+   :align: center
+   :scale: 50%
+
+License
+-------
+
+Usage of GLPI is subject to GNU license approval. Once licensing terms read an accepted, just validate the form.
+
+.. image:: images/license_agreement.png
+   :alt: Licensing terms
+   :align: center
+   :scale: 50%
+
+If you do not agree with licensing terms, it is not possible to continue installation process.
+
+Install / Update
+----------------
+
+This screen allows to choose between a fresh GLPI installation or an update.
+
+.. image:: images/install-update.png
+   :alt: Install or update
+   :align: center
+   :scale: 50%
+
+Click on install.
+
+Environment checks
+^^^^^^^^^^^^^^^^^^
+
+This step will check if prerequisites are met. If thery're not, it is not possible to continue and an explicit error message will tell you about what is wrong and what to do before trying again.
+
+.. image:: images/setup.png
+   :alt: Check prerequisites
+   :align: center
+   :scale: 50%
+
+Some prerequisites are optionals, it will be possible to continue installation event if thery're not met.
```

Comment:
optionals -> optional ? (not sure about this one)

Reply:
I guess there is a 's' since we're talking about several prerequisites